bundler/lib/bundler/man/bundle-exec.1.ronn

```diff
@@ -108,8 +108,8 @@ need to use `with_unbundled_env`.
 Bundler provides convenience helpers that wrap `system` and `exec`, and they
 can be used like this:
 
-    Bundler.clean_system('brew install wget')
-    Bundler.clean_exec('brew install wget')
+    Bundler.unbundled_system('brew install wget')
+    Bundler.unbundled_exec('brew install wget')
```

Comment:
Can you run also `rake man:build` and submit change to rendered file? Feel free to ping me if you need help. Alternatively I can do this and update the PR for you.

Reply:
Hm. I ran it and it touched every man page to change the date. I'm going to assume that's not desired, so I only added the bundle-exec one.

Reply:
great, thanks